bin/bootstrap-harmony

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# run harmony from scratch
+
+# check for docker
+if ! command -v "docker" &> /dev/null; then
+  echo "'docker' command not found. Harmony requires a running Docker environment."
+  exit 1;
+fi
+
+# check for kubectl command
+if ! command -v "kubectl" &> /dev/null; then
+  echo "'kubectl' command not found. Harmony requires a running Kubernetes cluster with the 'kubectl' command in your path."
+  exit 1;
+fi
+
+# read EDL credentials from .env file if it exists, otherwise prompt for them
+
+if [[ -f ".env" ]]; then
+  source .env
+else
+  ./bin/create-dotenv
+
+  # remove the empty OUATH env vars already in the file
+  awk -v n=3 'NR==FNR{total=NR;next} FNR==total-n+1{exit} 1' .env .env > env.tmp && mv env.tmp .env
+
+cat <<PROMPT
+Running Harmony requires creating an Earthdata Login (EDL) application. If you need instructions for this or do not understand what this means, please contact the Harmony team for help. Once you have your EDL application created, please enter the credentials as prompted.
+
+PROMPT
+
+  # prompt for the users EDL app credentials
+  read -p "EDL Application Client ID: " OAUTH_CLIENT_ID
+  echo ""
+  read -p "EDL Application UID: " OAUTH_UID
+  echo ""
+  read -s -p "EDL Application Password: " OAUTH_PASSWORD
+  echo ""
+
+cat << EOF >> .env
+OAUTH_CLIENT_ID=$OAUTH_CLIENT_ID
+OAUTH_UID=$OAUTH_UID
+OAUTH_PASSWORD=$OAUTH_PASSWORD
+EOF
+fi
+
+# run argo, localstack, and harmony in kubernetes
+./bin/start-all
```

Comment:
The `start-argo` script ends by telling you the URLs and ports for Argo and Localstack as well as how to delete it.  This script or `start-all` or somewhere should give equivalent information for localhost:3000 for Harmony.
May also want to suppress some of the output from starting Harmony to tidy the end of the script up so it's _just_ URLs and instructions

```
deployment.apps/harmony created
service/harmony created
```

Reply:
Done